### DISCUSSION TITLE
Wrong view on mobile

### DISCUSSION THREAD

**Discussion by IMC-GER** [fancyapps/ui#55] [Q&A]
Hi ,

I have a problem with the display of Fancybox 4.0.0beta0. 
On phones (Chrome/Android & Safari/iPhone) the images in the carousel overlap. (see screenshot)![Error_Smartphone](https://user-images.githubusercontent.com/88725484/128900323-68e9f099-f1f5-476f-9e64-608f6381749b.jpg)
In fullscreen mode it is shown correctly.

I suspect that the error is caused by the viewport when the initial-scale is smaller than 1.

Is there a way to open/start the fancybox in fullscreen mode?

**Comment by fancyapps** [comment]
Hi,

1) You are right, I can confirm that this issue appears when viewport scale is less than 1.
2) Sorry, there is currently no such option, but it will be available in the next update.

Thanks for your feedback!
